packages/create-contentful-app/lib/index.js

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console, no-process-exit */
+
+const spawn = require('cross-spawn');
+const path = require('path');
+const Spinner = require('cli-spinner').Spinner;
+const fetchAppDefinitionSettings = require('./prompt-app-definition');
+const createAppDefinition = require('./create-app-definition');
+
+const command = process.argv[2];
+const appFolder = process.argv[3];
+
+function onSuccess(spinner, folder) {
+  spinner.stop(true);
+
+  console.log();
+  console.log(`Success! Initialized "${folder}" in ~/${folder}.`);
+  console.log();
+  console.log(`We suggest that you begin by running:`);
+  console.log();
+  console.log(`   cd ${folder}`);
+  console.log(`   create-contentful-app create-definition`);
```

Comment:
should this say `npx create-contentful-app create-definition`?

Reply:
Yep - will be standardized for all commands after agreeing that `npx` is the preferred way to use this. Will be in the new repo